modules/adformBidAdapter.js

```diff
@@ -112,6 +116,12 @@ export const spec = {
           vastXml: response.vast_content,
           mediaType: type
         };
+
+        if (!bid.renderer && utils.deepAccess(bid, 'mediaTypes.video.context') === 'outstream') {
```

Comment:
@mike-chowla this if-test should have an extra condition: type === VIDEO

if (!bid.renderer && utils.deepAccess(bid, 'mediaTypes.video.context') === 'outstream' && type === VIDEO)

if you have a placement that has other mediaTypes (banner,native) than outstream it will add the rendrer either way, and cause the ad rendring to fail.

Reply:
@bratrahim it might be you that added this code and not @mike-chowla?

Reply:
@MonokkelFredrik You're right, that was me. Will prepare a pull request